build.gradle

```diff
@@ -6,15 +6,21 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.3.0'
-        // Exclude the version that the android plugin depends on.
-        configurations.classpath.exclude group: "com.android.tools.external.lombok"
+        classpath 'com.getkeepsafe.dexcount:dexcount-gradle-plugin:0.8.5'
     }
 }
 
 allprojects {
     repositories {
         google()
         jcenter()
+        maven { url "https://jitpack.io" }
+    }
+
+    gradle.projectsEvaluated {
+        tasks.withType(JavaCompile) {
+            options.compilerArgs << "-Xlint" << "-Xlint:-deprecation" << "-Xlint:-serial"
```

Comment:
I think there is a nicer way to ignore lint errors, instead of adding command line arguments. If you are already changing gradle files, could you please have a look at this? :)

Reply:
It's there to show you during compile what's wrong for given reasons. If you remove it, you will not pointed to it.
You can remove it, or show a smarter way too.
For me at least, it's usefull

Reply:
There is a `lintOptions` block. Does this do the same? Looks more like the "standard" way of enabling lint.

https://developer.android.com/studio/write/lint